skypy/supernova/Ia_redshift_rate.py

```diff
@@ -0,0 +1,45 @@
+r"""Creates the volumetric SNe Ia rate for an array of redshifts
+=================
+.. autosummary::
+   :nosignatures:
+   :toctree: ../api/
+   SNeIa_redshifts
+
+Models
+======
+.. autosummary::
+   :nosignatures:
+   :toctree: ../api/
+   Ia_redshift_rate
+"""
+
+from astropy import units
+
+
+def Ia_redshift_rate(redshift, r0=2.27e-5, a=1.7):
```

Comment:
Wondering what are these default values? Are they "standard" values? I would generally prefer them to not have defaults in most cases. They should also be documented under 'Parameters' even if it just says 'free parameters of the model'